internal/pkg/cli/env_delete_test.go

```diff
@@ -264,7 +264,7 @@ func TestDeleteEnvOpts_Execute(t *testing.T) {
     DeletionPolicy: Retain`, nil)
 				deployer.EXPECT().DeleteEnvironment(gomock.Any(), gomock.Any(), gomock.Any()).Return(errors.New("some error"))
 
-				prog.EXPECT().Stop(log.Serror("Failed to delete environment test from application phonetool."))
+				prog.EXPECT().Stop(log.Serror("Failed to delete environment test from application phonetool.\n"))
```

Comment:
can we make it consistent for using `Serror` and `Serrorln`?

Reply:
For `env_delete`, I used `\n` in the string consts because I can't do `ln` with `f` and those use `Serrorf` and `Ssuccessf`. So for this one, it's `fmtDeleteEnvFailed   = "Failed to delete environment %s from application %s.\n"`